Smallpatches

InfusOnWoW opened the pull request.

emptyrivers commented on WeakAuras/WeakAuras.lua:
```diff
@@ -204,26 +204,17 @@ local globalDynamicConditionFuncs = {};
 
 -- Unused regions that are kept around for clones
-WeakAuras.clonePool = {};
+Private.clonePool = {};
```
I've seen one aura in the past that this would be a breaking change for. But I don't think that's cause for concern.

InfusOnWoW replied:
Oh. 

Without looking at the code, I'm pretty sure that's a point in favor of moving it to private.

emptyrivers commented on WeakAurasOptions/BuffTrigger.lua:
```diff
@@ -90,20 +98,12 @@ local function GetBuffTriggerOptions(data, triggernum)
       order = 8.4,
       name = "",
-      hidden = function()
-        -- For those that update without restarting
```
Thank god we no longer have to do this

nullKomplex commented on WeakAurasOptions/BuffTrigger.lua:
```diff
@@ -56,11 +56,19 @@ local function getAuraMatchesList(name)
+    deleteNote = {
+      type = "description",
+      order = 8,
+      name = L["Note: The legacy buff trigger is now permantely disabled. It will be removed in the near future."],
```
> permantely 

permanently

Stanzilla commented on WeakAurasOptions/OptionsFrames/OptionsFrame.lua:
```diff
@@ -403,6 +411,117 @@ function WeakAuras.CreateFrame()
+
+  addFooter(L["Get Help"], [[Interface\AddOns\WeakAuras\Media\Textures\discord.tga]], "https://discord.gg/wa2",
+            L["Chat with WeakAuras Experts on our Discord Server."])
+  addFooter(L["Documentatiion"], [[Interface\AddOns\WeakAuras\Media\Textures\GitHub.tga]], "https://github.com/WeakAuras/WeakAuras2/wiki",
```
```suggestion
  addFooter(L["Documentation"], [[Interface\AddOns\WeakAuras\Media\Textures\GitHub.tga]], "https://github.com/WeakAuras/WeakAuras2/wiki",
```

Stanzilla commented on WeakAurasOptions/OptionsFrames/OptionsFrame.lua:
```diff
@@ -403,6 +411,117 @@ function WeakAuras.CreateFrame()
+  addFooter(L["Documentatiion"], [[Interface\AddOns\WeakAuras\Media\Textures\GitHub.tga]], "https://github.com/WeakAuras/WeakAuras2/wiki",
+            L["Read our Wiki."])
+  addFooter(L["Find Auras"], [[Interface\AddOns\WeakAuras\Media\Textures\wagoupdate_logo.tga]], "https://wago.io",
+            L["Browse the biggest collection of Auras"])
```
```suggestion
            L["Browse Wago, the largest collection of auras."])
```

Stanzilla commented on WeakAurasOptions/OptionsFrames/OptionsFrame.lua:
```diff
@@ -403,6 +411,117 @@ function WeakAuras.CreateFrame()
+  addFooter(L["Get Help"], [[Interface\AddOns\WeakAuras\Media\Textures\discord.tga]], "https://discord.gg/wa2",
+            L["Chat with WeakAuras Experts on our Discord Server."])
+  addFooter(L["Documentatiion"], [[Interface\AddOns\WeakAuras\Media\Textures\GitHub.tga]], "https://github.com/WeakAuras/WeakAuras2/wiki",
+            L["Read our Wiki."])
```
```suggestion
            L["Check out our wiki for a large collection of examples and snippets."])
```

Stanzilla commented on WeakAurasOptions/OptionsFrames/OptionsFrame.lua:
```diff
@@ -403,6 +411,117 @@ function WeakAuras.CreateFrame()
+
+  if not WeakAurasCompanion then
+    addFooter(L["Update Auras"], [[Interface\AddOns\WeakAuras\Media\Textures\wagoupdate_refresh.tga]], "https://weakauras.wtf",
+            L["Keep your wago imports up to date with the Companion App"])
```
```suggestion
            L["Keep your Wago imports up to date with the Companion App."])
```

Stanzilla commented on WeakAurasOptions/OptionsFrames/OptionsFrame.lua:
```diff
@@ -403,6 +411,117 @@ function WeakAuras.CreateFrame()
+            L["Keep your wago imports up to date with the Companion App"])
+  end
+  addFooter(L["Found a Bug?"], [[Interface\AddOns\WeakAuras\Media\Textures\bug_report.tga]], "https://github.com/WeakAuras/WeakAuras2/issues/new",
+            L["Report bugs in our ticketing system"])
```
```suggestion
            L["Report bugs our our issue tracker."])
```

Stanzilla commented on WeakAurasOptions/OptionsFrames/OptionsFrame.lua:
```diff
@@ -403,6 +411,117 @@ function WeakAuras.CreateFrame()
+  end
+
+  addFooter(L["Get Help"], [[Interface\AddOns\WeakAuras\Media\Textures\discord.tga]], "https://discord.gg/wa2",
+            L["Chat with WeakAuras Experts on our Discord Server."])
```
```suggestion
            L["Chat with WeakAuras experts on our Discord server."])
```

InfusOnWoW commented:
* fixed all the typos
* added some small amount of space at the top
* aligned left and right side
* replaced our search widget with the default search widget.
